### ISSUE TITLE
Build against the manifest previously downloaded instead of repository one

### ISSUE THREAD

**Issue by portellaa** [kubeflow/website#1700]
Hi,

I have a question about which "file" use to build the google cloud platform deployment. 

Shouldn't we use the manifest that we previously downloaded from the github and customized? 
I have been doing it like that and it seems to be working. From my interpretation from the built package, it is already configured using our values in the manifest instead of need to configure it again.

What i'm talking about is in the section [Customizing your kubeflow deployment](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/#customizing-your-kubeflow-deployment) in the step three.

Currently in the documentation it is like this 👇 
![Current documentation](https://user-images.githubusercontent.com/1502403/74735093-dd77e800-5247-11ea-9b4e-daadebae23c4.png)

But i think it should be like this 👇 
![What i think it should be](https://user-images.githubusercontent.com/1502403/74735124-ecf73100-5247-11ea-89d7-9bd4e425970b.png)

Please ignore this if i'm wrong 😊 

Cheers 🙏

**Comment by sarahmaddox** [comment]
Thanks @portellaa You are right. This is a copy-paste error. The line should be as you suggest:

```
kfctl build -V -f ${CONFIG_FILE}
```

**Comment by portellaa** [comment]
@sarahmaddox thank you. can i send a PR to fix this?

**Comment by sarahmaddox** [comment]
Yes please  :)
/assign @portellaa

**Comment by portellaa** [comment]
@sarahmaddox should i close the issue or is it up to you? 😊

**Comment by sarahmaddox** [comment]
@portellaa You can do the honours. :) 

A tip for next time: In your PR comment you can add wording like this:
```
fixes https://github.com/kubeflow/website/issues/1700
```

And then the bot will mark the issue as fixed when the PR is merged.

**Comment by portellaa** [comment]
thank you @sarahmaddox i did the fix #1700 to refer the link, but i didn't knew about that, i didn't read all the commands that the bot accepts 😇

thanks 🙏
